src/main/java/org/spongepowered/api/entity/player/User.java

```diff
@@ -126,6 +128,20 @@
     void setBedLocation(@Nullable Location location);
 
     /**
+     * Check whether this user has an operator status.
+     *
+     * @return Whether this user is an operator
+     */
+    boolean isOperator();
```

Comment:
I'm not sure about this operator get/set short-cut. There are other ways to check of operator status using the permission API, see https://github.com/SpongePowered/SpongeAPI/pull/285#issuecomment-68066983